init/sftpgo-docker.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=SFTPGo sftp server
+After=docker.service
+
+[Service]
+User=root
+Group=root
+WorkingDirectory=/etc/sftpgo
+Environment=PUID=1003
+Environment=GUID=1003
+EnvironmentFile=-/etc/sysconfig/sftpgo.conf
+ExecStartPre=-docker kill sftpgo
+ExecStartPre=-docker rm sftpgo
+ExecStart=docker run --name sftpgo \
```

Comment:
This way systemd will monitor the docker client and not the container. Anyway we can keep this service file but inside the docker/alpine folder since it is specific for this Dockerfile, for example it cannot be used with the debian Dockerfile I'm working on. Another option is to copy and paste it inside the README as example

Reply:
Yes yes, this init is only for docker with alpine multi-user